manifests/jetty.pp

```diff
@@ -50,26 +52,30 @@
   file { '/var/log/jetty':
     ensure => 'link',
     target => "${jetty_home}/jetty/logs",
-  } ->
-
-  file_line { 'jetty_startup_minutes':
-    ensure => present,
-    path   => "${jetty_home}/jetty-distribution-${jetty_version}/bin/jetty.sh",
-    line   => "    sleep ${jetty_start_interval}",
-    match  => '^    sleep \d+',
-  } ->
-
-  file { '/etc/init.d/jetty':
-    ensure => 'link',
-    target => "${jetty_home}/jetty-distribution-${jetty_version}/bin/jetty.sh",
   }
 
   if $::service_provider == 'systemd' {
     systemd::unit_file { 'jetty.service':
       content => template("${module_name}/jetty/jetty.service.erb"),
-      require => File['/etc/init.d/jetty'],
```

Comment:
Nothing in the archive is used to start the service?
This is a common ordering:
Install (archive)
Service (unit file -> service)